modules/raft/src/main/java/org/apache/ignite/raft/jraft/core/NodeImpl.java

```diff
@@ -889,7 +905,7 @@ public boolean bootstrap(final BootstrapOptions opts) throws InterruptedExceptio
         final long bootstrapLogTerm = opts.getLastLogIndex() > 0 ? 1 : 0;
         final LogId bootstrapId = new LogId(opts.getLastLogIndex(), bootstrapLogTerm);
         this.options = opts.getNodeOptions() == null ? new NodeOptions() : opts.getNodeOptions();
-        this.clock = options.getClock();
+        this.clock = this.options.getClock() == null ? new HybridClockImpl() : this.options.getClock();
```

Comment:
Currently clocks can't be null, because it's required for safe time generation. 
This logic is required for some tests, which are not setting clocks directly.